ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/http/JsonRpcErrorHandler.java

```diff
@@ -61,6 +62,12 @@ public void handle(final RoutingContext context) {
             requestId,
             statusCode,
             JsonRpcError.CONNECTION_TO_DOWNSTREAM_NODE_TIMED_OUT);
+      } else if (failure instanceof IllegalStateException
+          && statusCode == HttpResponseStatus.FORBIDDEN.code()) {
```

Comment:
Will TLS authentication failure end up here too?

Reply:
no, this is the only behavior that seems to have changed.